identity/src/main/java/com/android/identity/Util.java

```diff
@@ -198,6 +198,19 @@ byte[] cborEncode(@NonNull DataItem dataItem) {
         return baos.toByteArray();
     }
 
+    static @NonNull
+    byte[] cborEncodeNonCanonical(@NonNull DataItem dataItem) {
```

Comment:
nit: I'd prefer this be called something like `cborEncodePreserveOrder` or something.

Saying "NonCanonical" doesn't immediately imply that order is preserved. It just implies "I'm not canonicalizing". That is, it says what it _doesn't_ do. I think it's more helpful to tell me what the method _does_ do.

Maybe `cborEncodeWithoutCanonicalizing`? I dunno. Something more descriptive is all I ask for.

Reply:
Sure, `cborEncodeWithoutCanonicalizing` sounds better. Doing that. PTAL.